### PR TITLE
Release Candidate for L1 2021-11

### PR DESCRIPTION
**Pull request by glmxndr**

### CODE REVIEW

File: applications/processor/eisp/src/main/java/com/csgroup/s2pdgs/app/processor/RawToL0uFunction.java [hunk 1]
```diff
@@ -153,6 +154,9 @@ public record RawToL0uFunction(RawToL0uProperties props, ShellScriptLauncher lau
                 product.setSharedContextPath(sharedPath.toAbsolutePath().toString());
                 product.setRawPath(productRaw.getLocalPath());
                 product.setL0uPath(l0UPath.toAbsolutePath().toString());
+                // FIXME: better conversion between SatelliteId and S2Unit
+                product.setUnit(S2Unit.valueOf(productRaw.getSatelliteId().replace("S2", "")));
```
Comment (ilaborie) [inline applications/processor/eisp/src/main/java/com/csgroup/s2pdgs/app/processor/RawToL0uFunction.java:158]: (Reminder) apply the FIXME

File: common/auxip-ingestion/src/main/java/com/csgroup/s2pdgs/auxip/ingestion/AuxipFilesIngestion.java [hunk 2]
```diff
@@ -91,7 +91,14 @@ public void ingestExistingAuxFiles() throws Exception {
                     if (catalogResult.size() == 0) {
                         var outPath = Path.of(this.baseRepository, product.getName());
                         LOGGER.info("Ingesting {} to internal catalog : {}", file.toAbsolutePath(), outPath.toAbsolutePath());
-                        Files.copy(file, outPath,StandardCopyOption.REPLACE_EXISTING);
+                        Files.copy(file, outPath, StandardCopyOption.REPLACE_EXISTING);
+                        long initLength = file.toFile().length();
```
Comment (ilaborie) [inline common/auxip-ingestion/src/main/java/com/csgroup/s2pdgs/auxip/ingestion/AuxipFilesIngestion.java:95]: Maybe extract an helper function to check file length

File: applications/processor/eisp/src/main/java/com/csgroup/s2pdgs/app/processor/RawToL0uFunction.java (hunk 1 above)
Comment (glmxndr) [reply applications/processor/eisp/src/main/java/com/csgroup/s2pdgs/app/processor/RawToL0uFunction.java:167]: Not sure what the correct way to modify is yet. We will have to deal with other satellites than S2, and they may have more possible units, so the fixme is just there to flag that there i some work to do once we settle on the scope of use for this pdgs.

**Comment by glmxndr** [comment]
This is a partial update because not an actual release candidate for L1 production, but it is stable and has many fixes which may needed by others. Another rc branch will follow, but for now we can integrate these changes in develop.

**Comment by glmxndr** [comment]
The bulk of the changes concerns a naming change for two apps, so most of the PR is file name change.

**Comment by glmxndr** [comment]
Also, this PR provides an improved version of scr.sh, which should be installed. (`./scr.sh --install`)

**Pull request merged**